extensions/adobe/experience/analytics/context-data.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/analytics/context-data",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
```

Comment:
Can you have field-groups that are restricted to only be used when you have already added another field group?  Should this be changed to [https://ns.adobe.com/experience/analytics/experienceevent-all](https://github.com/adobe/xdm/blob/master/extensions/adobe/experience/analytics/experienceevent-all.schema.json) so you have to include the Adobe Analytics schema first before you can use it?

Reply:
I see what you are saying. You know what ? Let me move this property to the experienceevent-all field group. That way, it will always appear when someone adds the experienceevent-all field group to the schema.